File: api/src/main/java/dev/ulloasp/mlsuite/schema/review/application/dto/SchemaReviewLinkContextDto.java
package dev.ulloasp.mlsuite.schema.review.application.dto;

import java.util.List;

import dev.ulloasp.mlsuite.schema.application.dto.SchemaDto;
import dev.ulloasp.mlsuite.schema.application.dto.SchemaVersionDto;

public record SchemaReviewLinkContextDto(
        SchemaReviewOrganizationDto organization,
        SchemaDto schema,
        SchemaVersionDto schemaVersion,
        List<SchemaReviewRunListItemDto> runs) {
}

File: frontend/src/api/review/dtos/schema-review-link-context-dto.ts
/*
SPDX-License-Identifier: MIT
Copyright (c) 2025 Pablo Ulloa Santin
*/

import type { SchemaReviewRunListItemDto } from "./index";
import type { SchemaDto, SchemaVersionDto } from "../../schemas/dtos";

export type SchemaReviewLinkContextDto = {
  organization: { id: number; name: string };
  schema: SchemaDto;
  schemaVersion: SchemaVersionDto;
  runs: SchemaReviewRunListItemDto[];
};
